components/resources/brave_shields_strings.grdp

```diff
@@ -201,4 +201,20 @@
   <message name="IDS_BRAVE_SHIELDS_NOT_VALID_ADDRESS" desc="Label for link about invalid pattern for content setting page dialog to add a site">
     Wildcards are not allowed for Brave Shields.
   </message>
+
+  <message name="IDS_BRAVE_SHIELDS_ONBOARDING_LABEL_WITH_COMPANIES" desc="Label for the onboarding bubble UI, informing users about Brave Shields blocking company-specific trackers on a website, along with the number of additional trackers blocked.">
+    Brave Shields blocked <ph name="COMPANY_LIST">$1</ph> and <ph name="BLOCKED_COUNT">$2</ph> other trackers on <ph name="SITE_URL">$3</ph>
```

Comment:
nit: ideally we would use the pluralisation helper to make sure we say

"and 1 other tracker"
"and 2 other trackers"